server-common/src/main/java/org/apache/kafka/server/common/Features.java

```diff
@@ -40,7 +40,8 @@ public enum Features {
      * See {@link TestFeatureVersion} as an example. See {@link FeatureVersion} when implementing a new feature.
      */
     TEST_VERSION("test.feature.version", TestFeatureVersion.values()),
-    GROUP_VERSION("group.version", GroupVersion.values());
+    GROUP_VERSION("group.version", GroupVersion.values()),
+    KRAFT_VERSION("kraft.version", KRaftVersion.values());
```

Comment:
Should we add a test that shows that ApiVersionsResponse returns what we expect and it includes the kraft.version?

Reply:
Sounds good. Thanks!